README.md

```diff
@@ -43,7 +43,8 @@ Once up and running, access Solr's UI within your browser by opening
 `http://<projectname>.ddev.site:8983`. For example, if the project is named
 "myproject" the hostname will be `http://myproject.ddev.site:8983`.
 
-To access the Solr container from the web container use `http://ddev-<projectname>-solr:8983`.
+To access the Solr container from DDEV's web container, use either `http://ddev-<projectname>-solr:8983`
+or simply `http://solr:8983`.
```

Comment:
```suggestion
To access the Solr container from DDEV's web container, use  `http://solr:8983`.
```

There's no real need for the old fully-qualified hostname any more, as DDEV figures it out.

Reply:
Even better, and simpler